Commands/Public/room.js

```diff
@@ -0,0 +1,195 @@
+const ArgParser = require("../../Modules/MessageUtils/Parser");
+const moment = require("moment");
+
+module.exports = async ({ Constants: { Colors, Text }, client }, { serverDocument, serverQueryDocument }, msg, commandData) => {
+	const roomDocument = serverDocument.config.room_data.id(msg.channel.id);
+	if (roomDocument) {
+		const question = await msg.send({
+			embed: {
+				color: Colors.PROMPT,
+				description: `This room was created ${moment(roomDocument.created_timestamp).fromNow()}. Would you like to delete it?`,
+			},
+			footer: {
+				text: "You have 1 minute to respond.",
+			},
+		});
+		let response = (await msg.channel.awaitMessages(res => res.author.id === msg.author.id, { max: 1, time: 60000 })).first();
+		if (response) {
+			try {
+				await response.delete();
+			} catch (err) {
+				// No-op
+			}
+		}
+		if (response && configJS.yesStrings.includes(response.content.toLowerCase().trim())) {
+			let success = true;
+			await msg.channel.delete(`Room Management | Command issued by ${msg.author.tag}`).catch(err => {
+				success = false;
+				winston.debug(`Failed to delete room '${msg.channel.name}' on server '${msg.guild.name}'`, { svrid: msg.guild.id, chid: msg.channel.id, err });
+				msg.send({
+					embed: {
+						color: Colors.LIGHT_ERR,
+						title: "Failed to delete room!",
+						description: "I might be missing sufficient permissions!",
```

Comment:
```suggestion
						description: "Make sure I have the right permissions to do so!",
```

Reply:
Assumes we're talking to a Server Admin.

Reply:
Well I mean only server admins or the creator of the room should be able to delete it anyways... But I see what you mean

Reply:
As a side note, this may be a good reason to implement the same permission system as we use with tags in talk rooms. That features (such as deleting rooms, adding members) can be set to admin-only in the dashboard.